### DISCUSSION TITLE
Model query().where('Id',1) convert key to lowercase

### DISCUSSION THREAD

**Discussion by maydinbil61** [adonisjs/core#4004] [Help] [answered]
Merhaba SQL bağlantısı kullanıyorum. ORM üzerinden query().where() içersinde Id yazmana rağmen otomatik olarak id oluyor. Bu yüzden çalıştıramıyorum. Model Filed Name is Id.
const user1 = await TanimUser.query().where('Id', '=', 1).first()
 find end findby notworking

my Base Model

import { column, BaseModel } from '@ioc:Adonis/Lucid/Orm'

export default class AppBaseModel extends BaseModel {
  @column({ isPrimary: true })
  public Id: number
}

**Comment by RomainLanz** [comment]
Please write your message in English only.

**Comment by WeyIin** [comment]
Not exactly sure what you're after since I don't understand what you're asking but you can do 

`await TanimUser.find(1)`

**Comment by maydinbil61** [comment, marked as answer]
Hello SQL connection. Although you write Id in the query().where() content over the ORM, it is automatically id. That's why I can't run it. Model File filed name is ID.

**Reply by maydinbil61** [reply]
@column({ columnName: 'IP' })
